docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -507,6 +507,10 @@ tls:
      - Sets the size of the buffer used for reading the first part of a response received from the upstream server. See the `proxy_buffer_size <https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size>`_ directive. The default is set in the ``proxy-buffer-size`` ConfigMap key.
      - ``string``
      - No
+   * - ``use-cluster-ip``
+     - Enables using Cluster IP and port instead of the default behavior of using Pod IP and port. When this field is enabled, the fields that configure NGINX behavior related to multiple upstream servers (like `lb-method` and `next-upstream`) will have no effect, as the Ingress Controller will configure NGINX with only one upstream server that will match the service Cluster IP.
```

Comment:
```suggestion
     - Enables using the Cluster IP and port of the service instead of the default behavior of using the IP and port of the pods. When this field is enabled, the fields that configure NGINX behavior related to multiple upstream servers (like `lb-method` and `next-upstream`) will have no effect, as the Ingress Controller will configure NGINX with only one upstream server that will match the service Cluster IP.
```